secrets/.db_user_pw

```diff
@@ -1 +1 @@
-GFDgretg£"Dfdsf%^dfgdfdfg£"$REfdg!£654yhgfHGF@kdfgVBVCb
```

Comment:
docker secrets (without swarm) always disappoint me as you end up with files like this. Luckily this isn't actually anything important (i.e a password to a web accessible db (what kind of madness is this)). Changed my test instance pw anyway because I wear a tin hat.